data/user_data_base.ps2

```diff
@@ -199,6 +209,12 @@ function Get-MyName {
     return $nameTag.Value
 }
 
+function Get-Apps {
+    $instanceId = (New-Object System.Net.WebClient).DownloadString("http://169.254.169.254/latest/meta-data/instance-id")
+    $appsTag =  Get-EC2Tag | Where-Object {$_.ResourceId -eq $instanceId -and $_.Key -eq 'Apps'}
```

Comment:
We could special case `alyx-outpost` in outpost2, that can be done in outpost2 pr, until that is done though, we have to specifically add `alyx-outpost` to the list.